movement/movement.c

```diff
@@ -218,6 +218,26 @@ void movement_illuminate_led(void) {
     }
 }
 
+bool movement_default_loop_handler(movement_event_t event, movement_settings_t *settings) {
+    (void)settings;
+
+    switch (event.event_type) {
+        case EVENT_MODE_BUTTON_UP:
+            movement_move_to_next_face();
+            break;
+        case EVENT_LIGHT_BUTTON_DOWN:
+            movement_illuminate_led();
+            break;
+        case EVENT_MODE_LONG_PRESS:
```

Comment:
Probably could/should handle the MOVEMENT_SECONDARY_FACE_INDEX stuff here as well.

Reply:
Good call 👍